services/movie.py

```diff
@@ -0,0 +1,46 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+) -> QuerySet[Movie]:
+    queryset = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids,
+        )
+    elif genres_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+        )
+    elif actors_ids:
+        queryset = queryset.filter(
+            actors__id__in=actors_ids,
+        )
```

Comment:
refactor using two if statements: `if genres_ids:` and `if actors_ids:`. Get rid of `if genres_ids and actors_ids:` statemtent.